### DISCUSSION TITLE
termux on fire tablet KFONWI

### DISCUSSION THREAD

**Discussion by gitra123** [termux/termux-app#3807] [General]
i use termux on KFONWI, it's a good tablet to use termux?

**Comment by agnostic-apollo** [comment]
Device has 2gb ram and an entry level MediaTek MT8168 SOC, for basic commands will work fine, not for anything resource intensive.

https://developer.amazon.com/docs/fire-tablets/ft-device-specifications-firehd-models.html?v=firehd8_2020
